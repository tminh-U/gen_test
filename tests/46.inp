6
4 3 15 12 19 4 
15
1 4
1 3
4 6
5 6
3 3
4 5
1 2
6 6
6 6
3 6
1 3
3 6
5 6
4 5
3 4
