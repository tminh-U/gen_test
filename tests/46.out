9
6
5
3
2
4
3
2
2
7
6
7
3
4
3
